ci-operator/step-registry/ipi/aws/pre/ipi-aws-pre-chain.yaml

```diff
@@ -3,6 +3,12 @@ chain:
   steps:
   - chain: ipi-conf-aws
   - chain: ipi-install
+  env:
+    - name: COMPUTE_WORKER
+      default: 'null'
+      documentation: |-
+        The env var COMPUTE_WORKER defines type of flavour used for worker nodes, e.g. on AWS some tests may require larger m4.xlarge machines.
+        The default value "null" honours openshift-installer preset.
```

Comment:
This should go directly to the step (the `-ref.yaml` file), not to the chain. The step needs to be usable on it's own.

Reply:
I tried that in my initial commit https://github.com/openshift/release/pull/13712/commits/5b4f42369f947545fb811217c2074d276a13da68, but it failed other tests. Anyway let's wait for the outcome of DPTP-1740.